onnxruntime/core/optimizer/nchwc_transformer.cc

```diff
@@ -1023,7 +1024,7 @@ void NchwcTransformerImpl::TransformResize(Node& node) {
     scales_arg = input_defs[1];
   }
 
-  std::vector<int64_t> scales_attr(4);
+  TensorShapeVector scales_attr(4);
```

Comment:
>TensorShapeVector [](http://example.com/codeflow?start=2&length=17)

does `scales_attr` represent a tensor shape? it's not obvious to me but I'm not familiar with this code. if not, InlinedVector<int64_t, 4> could be an alternative

Reply:
I would just do InlinedVector then bc we do not want too many second dims to template proliferation.